api/src/main/java/com/theokanning/openai/completion/chat/ChatCompletionResult.java

```diff
@@ -0,0 +1,38 @@
+package com.theokanning.openai.completion.chat;
+import com.theokanning.openai.Usage;
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Object containing a response from the chat completions api.
+ */
+@Data
+public class ChatCompletionResult {
+
+    /**
+     * Unique id assigned to this chat completion.
+     */
+    String id;
+
+    /**
+     * The type of object returned, should be "chat.completion"
+     */
+    String object;
+
+    /**
+     * The creation time in epoch seconds.
+     */
+    long created;
+
+    /**
+     * A list of all generated completions.
+     */
+    List<ChatCompletionChoice> choices;
+
+    /**
+     * The API usage for this request.
+     */
+    Usage usage;
+
+}
```

Comment:
did you missed model param here ?
this is an sample response
```json
{
  "id": "chatcmpl-6peKOXv1MPTwR727Ib70CfcPYjrSc",
  "object": "chat.completion",
  "created": 1677767768,
  "model": "gpt-3.5-turbo-0301",
  "usage": {
    "prompt_tokens": 44,
    "completion_tokens": 35,
    "total_tokens": 79
  },
  "choices": [
    {
      "message": {
        "role": "assistant",
        "content": "根据维基百科，刘若英是1968年6月12日出生的，目前是53岁。"
      },
      "finish_reason": "stop",
      "index": 0
    }
  ]
}
```

Reply:
Good catch, I'll verify this when I get home :)